packages/x-components/src/views/home/Home.vue

```diff
@@ -402,7 +410,7 @@
             <!-- Recommendations -->
             <Recommendations v-if="!$x.query.search || $x.noResults" #layout="{ recommendations }">
               <BaseVariableColumnGrid
-                #default="{ item: result }"
+                v-slot="{ item: result }"
```

Comment:
I guess it was auto-fixed by eslint, we should keep with `#default`.
```suggestion
                #default="{ item: result }"
```